port/port_posix.cc

```diff
@@ -55,9 +55,12 @@ void InitOnce(OnceType* once, void (*initializer)()) {
 
 bool HasAcceleratedCRC32C() {
 #if (defined(__x86_64__) || defined(__i386__)) && defined(__GNUC__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
The warning is valid, because __get_cpuid can return 0 without storing anything in provided pointers.
See https://github.com/gcc-mirror/gcc/blob/master/gcc/config/i386/cpuid.h#L256.
The correct fix is:
```
  unsigned int eax, ebx, ecx, edx;
  if (!__get_cpuid(1, &eax, &ebx, &ecx, &edx))
	return false;
  return (ecx & (1 << 20)) != 0;
```

I cannot find this code upstream.

Reply:
Oh, cool. Thanks for looking at it.

Closing.

Reply:
Did you apply the fix?

Reply:
Ah. Sorry, I thought you meant that as the code wasn't in upstream, this change wasn't needed. Ok, I'll update now and PR again. Thanks for clearing it up.

Reply:
Looks like I can't submit against this close issue. Opened #16